client/signup/steps/plans/style.scss

```diff
@@ -1,45 +1,11 @@
-$plans-2023-small-breakpoint: 880px;
-$plans-2023-medium-breakpoint: 1340px;
-$plans-2023-large-breakpoint: 1500px;
-
-@mixin onboarding-2023-pricing-grid-plans-breakpoints {
-	max-width: 414px;
-
-	@media ( min-width: $plans-2023-small-breakpoint ) {
-		max-width: 860px;
-	}
-
-	@media ( min-width: $plans-2023-medium-breakpoint ) {
-		max-width: 1320px;
-	}
-
-	@media ( min-width: $plans-2023-large-breakpoint ) {
-		max-width: 1480px;
-	}
-}
-
-@mixin internal-plans-page {
-	max-width: 414px;
-
-	@media ( min-width: $plans-2023-small-breakpoint ) {
-		max-width: 860px;
-	}
-
-	@media ( min-width: $plans-2023-medium-breakpoint ) {
-		max-width: 1320px;
-	}
-
-	@media ( min-width: $plans-2023-large-breakpoint ) {
-		max-width: 1566px;
-	}
-}
```

Comment:
Moved these to a separate SCSS partial file, `client/my-sites/plan-features-2023-grid/_media-queries.scss`.